inc/css.php

```diff
@@ -22,7 +22,6 @@ function __construct() {
 	 * @param int $resolution The pixel resolution that this applies to
 	 */
 	function add_css($selector, $attributes, $resolution = 1920) {
-		$attribute_string = '';
```

Comment:
So I imagine this should actually be `$attribute_string  = array();`. Not initializing an array can trigger minor PHP warnings, so always safer to keep that initialized.